docs/core/diagnostics/diagnostic-libraries.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Diagnostic libraries
+description: An overview of the available diagnostic libraries in .NET.
+ms.date: 10/16/2023
+ms.topic: overview
+---
+
+# Diagnostic libraries
+
+Available as a set of .NET Extensions, various diagnostic libraries provide a set of APIs and tools that enable you to collect and analyze diagnostic information about your .NET applications. From summarizing exceptions, to determining resource utilization and monitoring, to exposing health checks, and providing probes for Kubernetes, the diagnostic packages provide a rich set of functionality that can be used to instrument your apps.
+
+The following NuGet packages are available:
+
+| NuGet package | Description |
+|--|--|
+| [📦 Microsoft.Extensions.Diagnostics.ExceptionSummarization](https://www.nuget.org/packages/Microsoft.Extensions.Diagnostics.ExceptionSummarization) | Provides a mechanism for retrieving exception summary information. |
+| [📦 Microsoft.Extensions.Diagnostics.ResourceMonitoring](https://www.nuget.org/packages/Microsoft.Extensions.Diagnostics.ResourceMonitoring) | Provides resource monitoring that measures processor and memory usage. |
+| [📦 Microsoft.Extensions.Diagnostics.HealthChecks.Common](https://www.nuget.org/packages/Microsoft.Extensions.Diagnostics.HealthChecks.Common) | Provides a standard health check implementation, as well as host application lifetime hooks. |
+| [📦 Microsoft.Extensions.Diagnostics.HealthChecks.ResourceUtilization](https://www.nuget.org/packages/Microsoft.Extensions.Diagnostics.HealthChecks.ResourceUtilization) | Provides resource utilization health checks. |
+| [📦 Microsoft.Extensions.Diagnostics.Probes](https://www.nuget.org/packages/Microsoft.Extensions.Diagnostics.Probes) | Provides support for Kubernetes environmental probes. |
+
+## Exception summarization
+
+When you're trying to generate meaningful diagnostic messages for exceptions, maintaining the inclusion of pertinent information can pose a challenge. The standard exception message often lacks critical details that accompany the exception, while invoking the <xref:System.Exception.ToString%2A?displayProperty=nameWithType> method yields an excess of state information.
```

Comment:
```maintaining the inclusion of pertinent information can pose a challenge```

as non native English reader I am struggling to understand what does it mean. Could we write it using simpler wording?

like (if I understood correctly) - `generate meaningful diagnostic messages for exceptions without leaking sensitive data can be challenging`